README.md

```diff
@@ -72,6 +74,35 @@ Source this environment in and provide the password when prompted:
 $ source .env
 ```
 
+## Jupyter notebooks
+Though Jupyter notebooks and lab are well suited to running Visier connector code, the use of OS-level environment variables may not be ideal for many users in this setting. For this purpose and since version `0.9.9`, the connector supports [dotenv](https://pypi.org/project/python-dotenv/) to facilitate a more dynamic switching of Visier authentication parameters.
+
+### Jupyter OAuth2 example
+In order to authenticate using OAuth, you will need a registered client application configuration in your tenant. Administrator-level privileges are required in order to register OAuth clients. 
```

Comment:
To authenticate with OAuth, you must first register an OAuth 2.0 client in Visier. Visier administrators can register OAuth clients.